config.json

```diff
@@ -341,7 +347,8 @@
           "description": "Builds and runs the tests for the given configuration.",
           "settings": {
             "Project": "src/tests.builds",
-            "MsBuildLogging":"/flp:v=normal;LogFile=build-tests.log"
+            "MsBuildLogging":"/flp:v=normal;LogFile=build-tests.log",
+            "Performance": "default"
```

Comment:
Don't we also have to add this for the above alias? (tests)

Reply:
thanks, will add it. though not sure if that one is still in use.